src/renderer/bootstrap.tsx

```diff
@@ -19,6 +19,17 @@ import { App } from "./components/app";
 import { LensApp } from "./lens-app";
 import { themeStore } from "./theme.store";
 
+/**
+ * If this is a development buid, wait a second to attach
+ * Chrome Debugger to renderer process
+ * https://stackoverflow.com/questions/52844870/debugging-electron-renderer-process-with-vscode
+ */
+async function attachChromeDebugger() {
+  if (isDevelopment) {
+    await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
We already have a helper function for this. (in `src/common/utils`
```suggestion
    await delay(1000);
```